editor/gui/scene_tree_editor.cpp

```diff
@@ -174,15 +174,40 @@ void SceneTreeEditor::_cell_button_pressed(Object *p_item, int p_column, int p_i
 		EditorDockManager::get_singleton()->focus_dock(NodeDock::get_singleton());
 		NodeDock::get_singleton()->show_groups();
 	} else if (p_id == BUTTON_UNIQUE) {
-		undo_redo->create_action(TTR("Disable Scene Unique Name"));
-		undo_redo->add_do_method(n, "set_unique_name_in_owner", false);
-		undo_redo->add_undo_method(n, "set_unique_name_in_owner", true);
-		undo_redo->add_do_method(this, "_update_tree");
-		undo_redo->add_undo_method(this, "_update_tree");
-		undo_redo->commit_action();
+		bool ask_before_revoking_unique_name = EDITOR_GET("docks/scene_tree/ask_before_revoking_unique_name");
+		revoke_node = n;
+		if (ask_before_revoking_unique_name) {
+			String msg = vformat(TTR("Revoke Unique Name for node \"%s\"?"), n->get_name());
```

Comment:
We only use Title Case for titles, but this appears to be the body of the dialog.
```suggestion
			String msg = vformat(TTR("Revoke unique name for node \"%s\"?"), n->get_name());
```